apps/web/screens/UniversitySearch/Comparison.tsx

```diff
@@ -0,0 +1,330 @@
+import React, { useEffect, useState } from 'react'
+import getConfig from 'next/config'
+
+import {
+  Box,
+  Button,
+  GridContainer,
+  LinkV2,
+  Table as T,
+  Text,
+} from '@island.is/island-ui/core'
+import {
+  GetNamespaceQuery,
+  GetNamespaceQueryVariables,
+  GetUniversityGatewayQuery,
+  GetUniversityGatewayQueryVariables,
+  GetUniversityGatewayUniversitiesQuery,
+  UniversityGatewayProgramDetails,
+  UniversityGatewayUniversity,
+} from '@island.is/web/graphql/schema'
+import { useLinkResolver, useNamespace } from '@island.is/web/hooks'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import { Screen } from '@island.is/web/types'
+import { CustomNextError } from '@island.is/web/units/errors'
+
+import { GET_NAMESPACE_QUERY } from '../queries'
+import {
+  GET_UNIVERSITY_GATEWAY_PROGRAM,
+  GET_UNIVERSITY_GATEWAY_UNIVERSITIES,
+} from '../queries/UniversityGateway'
+import { TranslationDefaults } from './TranslationDefaults'
+
+const { publicRuntimeConfig = {} } = getConfig() ?? {}
+
+interface UniversityComparisonProps {
+  data: Array<UniversityGatewayProgramDetails>
+  locale: string
+  namespace: Record<string, string>
+  universities: Array<UniversityGatewayUniversity>
+}
+
+const Comparison: Screen<UniversityComparisonProps> = ({
+  data,
+  locale,
+  namespace,
+  universities,
+}) => {
+  const n = useNamespace(namespace)
+  const { linkResolver } = useLinkResolver()
+  const [selectedComparison, setSelectedComparison] =
+    useState<Array<UniversityGatewayProgramDetails>>(data)
+
+  const handleDelete = (dataItem: UniversityGatewayProgramDetails) => {
+    const found = selectedComparison.some((x) => x.id === dataItem.id)
+
+    if (found) {
+      const a = selectedComparison.filter((item) => {
+        if (item.id !== dataItem.id) {
+          return true
+        }
+        return false
+      })
```

Comment:
Isn't this whole thing just simply `.filter(item => item.id !== dataItem.id)` or will this receive additional logic later?

Reply:
yes it is, I change it